sim/dex.ts

```diff
@@ -325,21 +325,21 @@ export class ModdedDex {
 	}
 
 	/**
-	 * Convert a pokemon name, ID, or template into its species name, preserving
-	 * form name (which is the main way Dex.getSpecies(id) differs from
-	 * Dex.getTemplate(id).species).
+	 * Convert a pokemon name, ID, or species into its species name, preserving
+	 * form name (which is the main way Dex.getSpeciesName(id) differs from
+	 * Dex.getSpecies(id).species).
 	 */
-	getSpecies(species: string | Template): string {
-		const id = toID(species || '');
-		const template = this.getTemplate(id);
-		if (template.otherForms && template.otherForms.includes(id)) {
-			const form = id.slice(template.species.length);
-			if (form) return template.species + '-' + form[0].toUpperCase() + form.slice(1);
-		}
-		return template.species;
+	getSpeciesName(speciesid: string | Species): string {
```

Comment:
Per Discord, Zarel says this can be `getCosmeticFormeName`, and `Pokemon.speciesName` can be changed to `Pokemon.cosmeticFormeName`